### ISSUE TITLE
what are the hyperparameters under the setting of triplet+softmax?

### ISSUE THREAD

**Issue by LibreAgent2025** [lulujianjie/person-reid-tiny-baseline#10]
I appreciate your sententious work. However, I failed to reproduce your result under the setting of triplet+softmax, I thought it may result from the hyperparameters ( I only change COS_LAYER->no, LOSS_TYPE->triplet+softmax, TEST_METHOD->euclidean ). So, I hope you can provide the hyperparameters under the setting of triplet+softmax. Thank you!

**Comment by lulujianjie** [comment]
Hi, I have updated the hyperparameters in v0.1.2.
